docs/app/docs/jetify_cloud/deploys/setup.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Setting up Your Project
+sidebar_position: 2
+---
+
+## Configuring your Project
+
+Deploying with Jetify Cloud requires the following:
+
+1. **A Dockerfile**. You can generate a starter dockerfile for your project by running `devbox generate dockerfile`. Devbox will automatically build and run your Dockerfile whenever you push a change to your repo. Make sure that your application code is copied into the container, and that you add a command to start your service. 
```

Comment:
Dockerfile is no longer required, but maybe the install, build and start hooks in the future?